util/check-format-test-negatives.c

```diff
@@ -1,7 +1,6 @@
 /*
- * Copyright 2007-2021 The OpenSSL Project Authors. All Rights Reserved.
- * Copyright Nokia 2007-2019
```

Comment:
??? Should this really be removed???

Reply:
I anticipated this question :wink: 
The answer is yes - when I committed the first version of this file, I had mistakenly copied this line from a source file of the CMP contribution, where Nokia had been originally involved.
Yet in the tool `check-format` tool and its test cases Nokia was not involved.

Reply:
Cool, all good.